*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to 127.0.0.1:33293 (IPv4)
Extension host agent listening on 33293

[10:40:13] 




[10:40:13] Extension host agent started.
[10:40:13] [127.0.0.1][4e567614][ManagementConnection] New connection established.
[10:40:13] [127.0.0.1][966fd2e9][ExtensionHostConnection] New connection established.
[10:40:13] [127.0.0.1][966fd2e9][ExtensionHostConnection] <1671> Launched Extension Host Process.
New EH opened, aborting shutdown
[10:45:13] New EH opened, aborting shutdown
[10:59:55] No ptyHost heartbeat after 6 seconds
[11:31:36] No ptyHost heartbeat after 6 seconds
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[12:32:42] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[12:32:42] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[12:32:44] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[12:32:45] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[12:32:46] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[12:37:12] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
[12:46:32] [127.0.0.1][4e567614][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[12:46:32] [127.0.0.1][966fd2e9][ExtensionHostConnection] <1671> Extension Host Process exited with code: 0, signal: null.
Last EH closed, waiting before shutting down
[12:46:32] Last EH closed, waiting before shutting down
Got delay-shutdown request while in shutdown timeout, delaying
[12:46:48] Got delay-shutdown request while in shutdown timeout, delaying
Cancelling previous shutdown timeout
[12:46:48] Cancelling previous shutdown timeout
[12:46:48] [127.0.0.1][b3b61062][ManagementConnection] New connection established.
[12:46:48] [127.0.0.1][eb1409cd][ExtensionHostConnection] New connection established.
[12:46:48] [127.0.0.1][eb1409cd][ExtensionHostConnection] <17322> Launched Extension Host Process.
New EH opened, aborting shutdown
[12:51:48] New EH opened, aborting shutdown
[13:31:35] [File Watcher (node.js)] Watcher shutdown because watched path got deleted
[13:39:45] No ptyHost heartbeat after 6 seconds
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[13:58:13] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[14:08:20] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[14:08:21] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[14:08:24] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[14:08:25] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
[14:49:48] No ptyHost heartbeat after 6 seconds
[14:52:27] [127.0.0.1][6e3ff609][ManagementConnection] New connection established.
[14:52:27] [127.0.0.1][2fd5bdab][ExtensionHostConnection] New connection established.
[14:52:27] [127.0.0.1][2fd5bdab][ExtensionHostConnection] <26870> Launched Extension Host Process.
[14:57:28] [127.0.0.1][eb1409cd][ExtensionHostConnection] <17322> Extension Host Process exited with code: 0, signal: null.
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[16:15:03] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[16:24:43] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[16:24:43] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
[16:36:04] No ptyHost heartbeat after 6 seconds
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5)
[16:40:36] AggregateError
    at internalConnectMultiple (node:net:1081:18)
    at afterConnectMultiple (node:net:1532:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:8000
        at afterConnectMultiple (node:net:1519:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 8000
    }
  ]
}
